ngraph_bridge/ngraph_encapsulate_impl.cc

```diff
@@ -576,6 +580,36 @@ void NGraphEncapsulateImpl::DumpNgFunction(
   StringToFile(file_name, m_serialized_ng_function_map[ng_exec]);
 }
 
+Status NGraphEncapsulateImpl::GetPersistentTFOutputTensor(
+    std::shared_ptr<ngraph::runtime::Executable> exec,
+    std::vector<tensorflow::PersistentTensor>& tf_output_tensors) {
+  auto itr = m_out_persistents.find(exec);
+  if (itr == m_out_persistents.end()) {
```

Comment:
Could use PersistentOutputsExist

Reply:
Then I'd be doing 2 searches. One in `PersistentOutputsExist` and one in again to actually return the item. Right now I do only one search to find the appropriate iterator